docs-rtd/reference/upgrading-provider.md

```diff
@@ -0,0 +1,235 @@
+# Upgrading to Juju Terraform Provider v1.0.0
+
+This guide outlines the breaking changes and upgrade procedures when migrating from pre-v1.0.0 versions of the Juju Terraform provider to v1.0.0.
+
```

Comment:
I feel this is a mix of release notes content and a how-to. I'd refactor that way. 

PS In general it's a good idea to chat with me before doing any docs work, as it would give us a chance to discuss what needs to be done.

Reply:
Moved to a nested folder in the how-to.